project-base/storefront/components/Blocks/Login/LoginForm.tsx

```diff
@@ -80,6 +96,22 @@ export const LoginForm: FC<LoginFormProps> = ({
         );
     };
 
+    const SocialNetworkLoginLink: FC<{ socialNetwork: TypeLoginTypeEnum }> = ({ socialNetwork }) => {
+        return (
+            <ExtendedNextLink
+                className="bg-white h-12 w-full shadow-md rounded flex justify-center items-center hover:shadow-greyLight"
+                href={getSocialNetworkLoginUrl(
+                    socialNetwork,
+                    cartUuid,
+                    shouldOverwriteCustomerUserCart,
+                    productListUuids,
+                )}
+            >
+                {getSocialNetworkIcon(socialNetwork)}
+            </ExtendedNextLink>
+        );
+    };
+
```

Comment:
Defining components inside the scope of other components is considered antipattern (the component gets recreated on each render instead of re-rendering it only when its properties change). It is best practice to define them outside of the scope and provide required data through props. You could even consider splitting them into their own files and making them reusable, but I don't know if this applies to this case.

Reply:
:ok_hand: fixed in 98beaf69d1